quic/s2n-quic-transport/src/ack.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+pub mod interest {
+    #[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord)]
+    pub enum Interest {
+        None,
+        #[allow(dead_code)]
+        Immediate,
```

Comment:
I'm not sure we'll ever have any variants outside of this. Either it wants to do some ACK processing or it doesn't. In the case of delayed ACK processing, we would set a timer, and when that expires we'd return immediate.

Reply:
Is there ask from this comment? Maybe rename `Immediate` to `Some`?

Yep, was thinking about delays(threshold or time based) and hoping to capture it as a variant. But either of those requirements can/should be handled outside of the interest lists.